### PR TITLE
add an apache httpd error log format

### PR DESCRIPTION
**Pull request by igalic**
HTTPD_ERRORLOG matches either apache httpd 2.0/2, or 2.4

### CODE REVIEW

**Comment by purbon** [comment]
Thanks a lot for your contribution @igalic, I will be reviewing it.

File: patterns/grok-patterns [hunk 1]
```diff
@@ -89,6 +90,10 @@ QS %{QUOTEDSTRING}
 SYSLOGBASE %{SYSLOGTIMESTAMP:timestamp} (?:%{SYSLOGFACILITY} )?%{SYSLOGHOST:logsource} %{SYSLOGPROG}:
 COMMONAPACHELOG %{IPORHOST:clientip} %{USER:ident} %{USER:auth} \[%{HTTPDATE:timestamp}\] "(?:%{WORD:verb} %{NOTSPACE:request}(?: HTTP/%{NUMBER:httpversion})?|%{DATA:rawrequest})" %{NUMBER:response} (?:%{NUMBER:bytes}|-)
 COMBINEDAPACHELOG %{COMMONAPACHELOG} %{QS:referrer} %{QS:agent}
+HTTPD20_ERRORLOG \[%{HTTPDERROR_DATE:timestamp}\] \[%{LOGLEVEL:loglevel}\] (?:\[client %{IPORHOST:clientip}\] ){0,1}%{GREEDYDATA:errormsg}
+HTTPD24_ERRORLOG \[%{HTTPDERROR_DATE:timestamp}\] \[%{WORD:module}:%{LOGLEVEL:loglevel}\] \[pid %{POSINT:pid}:tid %{NUMBER:tid}\]( \(%{POSINT:proxy_errorcode}\)%{DATA:proxy_errormessage}:)?( \[client %{IPORHOST:client}:%{POSINT:clientport}\])? %{DATA:errorcode}: %{GREEDYDATA:message}
+HTTPD_ERRORLOG %{HTTPD20_ERRORLOG}|%{HTTPD24_ERRORLOG}
```
Comment (purbon) [inline patterns/grok-patterns]: wondering if there are still out there older versions of httpd, do you think we should somehow label it also with being 2.x branch?

**Comment by purbon** [comment]
Just left you a minor comment, but in general this looks great!

File: patterns/grok-patterns (hunk 1 above)
Comment (igalic) [inline patterns/grok-patterns]: i sure hope not ;)
we've deprecated 1.3 six(?) years ago.
we've deprecated 2.0 three years ago: http://www.gossamer-threads.com/lists/apache/dev/402809

Comment (purbon) [inline patterns/grok-patterns]: cool, thanks a lot for the education!

**Pull request closed without merging**